docs/source/apps.rst

```diff
@@ -126,6 +126,17 @@ Applications
 .. autoclass:: TileOnGridd
     :members:
 
+.. automodule:: monai.apps.pathology.transforms.post.array
+.. autofunction:: _coords_to_pixel
+.. autofunction:: _dist_from_topleft
+.. autofunction:: _sp_contours_to_cv
```

Comment:
You don't need to add the private functions to the documents. Thanks

Reply:
Removed it, thanks!

Reply:
Removed it, thanks!